src/mixins/_button.scss

```diff
@@ -17,6 +17,9 @@
 @include theme.props() {
   --theme-color-background-button-default: #{color.$brand-primary};
   --theme-color-background-button-secondary: transparent;
+  --theme-color-background-button-icon-slash-default: #{color.$brand-primary};
+  --theme-color-background-button-icon-slash-secondary: transparent;
+  --theme-color-background-button-icon-slash-tertiary: transparent;
```

Comment:
I'm unsure of the best way to handle this. The value of `transparent` doesn't actually do what I want it to. You'll notice that the extra slash is not visible in the secondary and tertiary buttons.

<img width="1252" alt="Screen Shot 2021-06-09 at 4 29 48 PM" src="https://user-images.githubusercontent.com/459757/121442307-3806ed00-c940-11eb-8c12-fdb50914f74a.png">

My attempted solution before this was using `clip-path` with a custom `polygon` but there was a [Safari issue I was unable to resolve](https://cloudfour.slack.com/archives/C04AEEVK2/p1623178062068100).

Any ideas? 🤔

Reply:
@gerardo-rodriguez What do you think about a solution where there is a variant of each slashable icon with the slash in it (rather than adding the slash in CSS)? That way the slash is positioned nicely even for icons where the ideal position of the slash is not centered

Reply:
@calebeby Good idea! I was trying to create a flexible solution that didn't require multiple SVG icons but if this solution isn't up-to-par, then having two SVG icons may be the only way to handle it. 

I'll be curious how our design team feels about this. 🔍 👀

Reply:
Hmm... 🤔 

Let me try something and get back to you.

Reply:
I'm also curious how the design team feels about it!

Hand-rolled slashes might look nicer in certain contexts but would be more work.

>I'm unsure of the best way to handle this. The value of transparent doesn't actually do what I want it to. You'll notice that the extra slash is not visible in the secondary and tertiary buttons.

Hmm, should the extra slash's color be set to the background's current background? (I guess if the button has a transparent background that wouldn't work...)

I'm not opposed to revisiting the clip-path solution if this isn't working well. If we did that I wonder if there's a way to share SVG code between the clip path and the visible stroke? If that's possible we could DRY things up, and ensure the primary stroke and secondary stroke render the same way.

Reply:
Okay, a few thoughts after poking around in this...

1. I'm curious if you ever tried using flex positioning instead of `inline-block` for the icon and if that would have resolved the Safari inconsistency?
2. I tried to experiment with using an SVG for this in a dynamic way, but I wasn't very happy with the results since they would require managing `defs` with `id` attributes that don't conflict: https://codepen.io/tylersticka/pen/783b2974d446edf1a8db02652e73afef
3. Supporting on/off icons is fine, but I'm really skeptical about `clip-path` not working when it did on a recent client project… 🤔

Reply:
> Supporting on/off icons is fine, but I'm really skeptical about clip-path not working when it did on a recent client project… 🤔

I was confused by this as well! 😳 And that's why [I asked for help](https://cloudfour.slack.com/archives/C04AEEVK2/p1623178062068100), I wasn't sure if I was doing something incorrectly. 😅 

I'm going to take another look. It seems silly that it worked before but wasn't working for me this time. ¯\\\_(ツ)_/¯

> I'm curious if you ever tried using flex positioning instead of `inline-block` for the icon and if that would have resolved the Safari inconsistency?


I can look into this, I'm not sure I tried this. 🔍 

Thank you! 😄

Reply:
LOL, never mind. I started a Codepen and realized the clip paths you want are opposites for the two strokes.

Reply:
Anyways, here's my hacky codepen in case it's helpful: https://codepen.io/phebert/pen/ZEeMwWm

Reply:
> LOL, never mind. I started a Codepen and realized the clip paths you want are opposites for the two strokes.

@Paul-Hebert Can you elaborate what you mean by "are opposite for the two strokes"? 🤔

Reply:
@gerardo-rodriguez I think what @Paul-Hebert means is that you can't use the stroke as a mask for the icon because that will mask the icon _within_ the stroke. What we actually need is to mask everything _outside_ of the stroke.

Reply:
Thanks, @tylersticka, I see.

I'm having a very hard time figuring this out. [This is my latest attempt](https://github.com/cloudfour/cloudfour.com-patterns/pull/1298/commits/cedd9e08785294cb9ece0b572a8dfa1bb124941f) but it's now harder for me to figure out the proper shape of the slash and Safari still renders differently from other browsers. 😕 

https://deploy-preview-1298--cloudfour-patterns.netlify.app/?path=/docs/components-button--toggling-button

Is anyone available to jump on a call? I am spinning my wheels. 😞

Reply:
Thanks for all your help, @Paul-Hebert, and @tylersticka!

I've pushed up my latest proposal, here's what it looks like:

<img width="1273" alt="Screen Shot 2021-06-11 at 4 10 45 PM" src="https://user-images.githubusercontent.com/459757/121756794-2fddb780-cad0-11eb-8bf4-3137cdcda712.png">

I ended up implementing a more _hacky_ solution to get Safari to render closer to the design intention (using `-webkit` prefix). The `supports` `view-box` route didn't produce the expected results for me, unfortunately.

Reply:
This is good progress, Gerardo. Unfortunately, I'm still noticing some odd visual issues with this approach. There are subpixel artifacts around the slash, and the discrepancy between the slash size and the shadow size is pretty significant.

I'm going to come back with some clearer direction of what we should try next so you know where to go from here.

Reply:
@gerardo-rodriguez I submitted a PR with a prototype approach that gets us to something I feel is a bit closer: #1302 

Happy to chat more on Monday!

Reply:
Thanks so much, @tylersticka!

I've updated this PR with the latest changes! 😄